imap_processing/tests/lo/test_bit_decompression.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from imap_processing.lo.l0.utils.bit_decompression import Decompress, decompress_int
+
+
+def test_decompress_int_8_to_16_bit():
+    """Test decompressing integer from 8 to 16 bits."""
+    ## Arrange
+    decompression = Decompress.DECOMPRESS8TO16
+    compressed_value = 209
+    expected_decompressed_int = 9537
```

Comment:
I'd like to see more parameterization in the tests to avoid the repetition if possible. For example, these cases could be combined here:
```suggestion
@pytest.mark.parameterize(("decompression", "compressed_value", "expected_decompressed_int"),
    [(Decompress.DECOMPRESS8TO16, 209, 9537),
      (Decompress.DECOMPRESS8TO12, 205, 1229),
      (Decompress.DECOMPRESS12TO16, 4068, 62908),
       ])
def test_decompress_int(decompression, compressed_value, expected_decompressed_int):
    """Test decompressing integer from 8 to 16 bits."""
```